config/openshift/namespace.yaml

```diff
@@ -10,5 +10,5 @@ metadata:
     openshift.io/node-selector: ""
     openshift.io/description: "Openshift ingress node firewall components"
     workload.openshift.io/allowed: "management"
-  name: system
+  name: openshift-ingress-node-firewall
```

Comment:
shouldnt this be replaced by kustomize anyway? You found it wasn't being replaced?

Reply:
Ns wasn’t created when ran on ocp I thought kustomize will use this yaml to create namespace if my understanding is incorrect let me know